manifests/pyvenv.pp

```diff
@@ -46,15 +46,8 @@
     # Debian splits the venv module into a seperate package
     if ( $facts['os']['family'] == 'Debian') {
       $python3_venv_package = "python${normalized_python_version}-venv"
-
-      case $facts['os']['distro']['codename'] {
-        'xenial', 'bionic', 'cosmic', 'disco', 'stretch', 'buster': {
-          ensure_packages($python3_venv_package)
-
-          Package[$python3_venv_package] -> File[$venv_dir]
-        }
-        default: {}
-      }
+      ensure_packages($python3_venv_package)
+      Package[$python3_venv_package] -> File[$venv_dir]
```

Comment:
Could you readd the empty line between `ensure_packages` and the sorting?

Reply:
Added empty line